CHANGELOG.md

```diff
@@ -3,13 +3,13 @@
 - `vweb` now uses struct embedding: `app.vweb.text('hello') => app.text('hello')`.
 - Consts can now be declared outside of `const()` blocks: `const x = 0`.
 - Allow overloading of  `>`, `<`, `!=` and `==` operators.
+- Add struct updating syntax: `Mystruct{ ...ms, name: 'new' }`.
```

Comment:
Should also add `deprecating MyStruct{ms | name: 'new'}`?

Reply:
Not yet. :D
Have to do it.